enhancements/installer/aws-outpost-remote-workers.md

```diff
@@ -0,0 +1,209 @@
+
+## Motivation
+
+Users might want to use OCP on AWS Outpost running in their own data center.
+AWS Outposts enables customers to build and run applications on premises using
+the same programming interfaces as in AWS Regions, while using local compute
+and storage resources for lower latency and local data processing needs.
+
+### User Stories
+
+Phase 0:
+- As a cluster admin I want to follow OCP documentation to deploy OCP on AWS Outpost
+  where my master nodes are running in the regular AWS region and workers in AWS Outpost.
+
+Phase 1:
+- As a cluster admin I want to use an OCP installer to deploy OCP on AWS Outpost.
+  (Cluster deployment topology depends on mitigating lack of support for NLB)
+
+Phase 2:
+- As a cluster admin I want to use ROSA cli or UI to deploy OCP/Hypershift on AWS Outpost.
+  (OCP deployment topology similar as in Phase 1)
+
+
+### Goals
+
+Phase 0:
+- Admin can follow documentation to customize installer generated manifests to deploy
+  OCP on AWS Outpost
+- As post installation step admin can configure ALB on AWS Outpost as workload ingress
+- Admin can check list of limitations or not supported features
+
+Phase 1:
+- Admin can use OCP installer to deploy OCP on AWS Outpost
+- After installation only gp2 storageclass is available and ingress is configured
+
+Phase 2:
+- Admin can use ROSA cli or UI to deploy OCP/Hypershift on AWS Outpost
+
+### Non-Goals
+
+We do not intend to implement AWS Outpost specific features in OCP as well us we
+do not plan to handle long lasting networking issues between regular AWS region andan outpost.
+
+## Proposal
+
+Phase 0:
+We want to provide documented manual procedure on how to deploy an OCP cluster in a regular
+AWS region with worker nodes running in AWS Outpost. The user needs to create VPC and
+subnets before the installation and provide them in the install-config to prevent VPC
+creation by IPI installer. The subnets need to be created both in the regular region and
+in the outpost (tagged as "kubernetes.io/cluster/<non-cluster-name>": "owned"). In the
+install-config only AWS region subnet needs to be used so NLB is created for the
+apiserver to use. During manifest stage it is needed to change workers' machineSet
+to use subnets created in the outpost. As the last modification the user needs to
+update MTU for the network provider due to AWS Outpost supporting only 1300 bytes. Once the
+cluster is deployed the user needs to install AWS load balancer operator and configure
+ALB ingress.
+
+Phase 1:
+Separate enhacement needed
+
+Phase 2:
+Separate enhacement needed
+
+### Workflow Description
+
+Phase 0:
+In order to start the cluster installation process the user needs to follow steps to
+configure AWS account and AWS Outpost instance types. Next the user needs to create
+VPC with subnets for both the AWS region and the outpost. The user needs to create
+cloud formation template with AWS outpost details and use aws client to run it.
```

Comment:
Is this "Setup your AWS Outpost" work or is this "Setup your AWS Outpost to work with OpenShift" work? If it's the former do we need to mention it?